image/templates/helm/stackrox-central/templates/01-central-15-exposure.yaml

```
@@ -41,6 +41,35 @@ spec:
     kind: Service
     name: central
 ---
+
+{{- if and ._rox.central.exposure.route.reencrypt.enabled ._rox._ca }}
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: central-reencrypt
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "srox.labels" (list . "route" "central-reencrypt") | nindent 4 }}
+  annotations:
+    {{- include "srox.annotations" (list . "route" "central-reencrypt") | nindent 4 }}
+    "helm.sh/hook": "pre-install,pre-upgrade"
+    "helm.sh/resource-policy": "keep"
+spec:
+{{- if ._rox.central.exposure.route.reencrypt.host }}
+  host: {{ ._rox.central.exposure.route.reencrypt.host }}
+{{- end }}
+  port:
+    targetPort: https
+  tls:
+    destinationCACertificate: |
+      {{- ._rox._ca.Cert | nindent 6 }}
+    termination: reencrypt
```

Is it possible to add support for custom TLS certificates?
This could be useful for the cloud service.

```suggestion
    termination: reencrypt
    {{- with ._rox.central.exposure.route.reencrypt.tls }}
    key: {{ .key }}
    certificate: {{ .certificate }}  
    {{- end }}
```
**Haven't tested it, use at your own risk** :)